docs/guides/admin/docs/modules/studio.md

```diff
@@ -14,6 +14,17 @@ The path `/studio` is accessible by users with the role `ROLE_ADMIN` or `ROLE_ST
 
 The preferred way to let your users access Studio is via LTI. Remember to configure your LTI users to have the role `ROLE_STUDIO` so that they can access Studio and all APIs used by Studio.
 
+This can be done for example by configuring the `LtiLaunchAuthenticationHandler` in `etc/org.opencastproject.security.lti.LtiLaunchAuthenticationHandler.cfg`. At the bottom of this file you see the two options `lti.custom_role_name` and `lti.custom_roles`. The former should be set to an LTI role that you want to give access to Studio to, and the latter is a comma-separated list of Opencast roles that users with this LTI role should get. This list has to at least include `ROLE_STUDIO`.
+
+A minimal configuration giving LTI-`Instructor`-s access to Opencast Studio could look like this:
+
+```
+lti.custom_role_name=Instructor
+lti.custom_roles=ROLE_STUDIO
+```
+
+Note that with this configuration, every LTI user is persisted in the database with their ID and all the roles assigned to them!
```

Comment:
Maybe mention that those users are not persisted in the normal user table, but merely in the user references table?

Reply:
Mentioned.